content/apps/using-oauth2.md

```diff
@@ -0,0 +1,157 @@
+# Using OAuth 2.0 to connect an App
+
+The PIM uses OAuth 2.0 to manage the authorization of Apps.
+
+![App activation diagram](../img/apps/app-activation-sequence-diagram.png)
+
+
+## Credentials
+
+To begin, you must obtain valid OAuth 2.0 client credentials by registering your App on the
+[Akeneo Marketplace](https://marketplace.akeneo.com/node/add/extension).
+
+Then, when any user want to connect his PIM to your App, the authorization process will go through the following steps.
+
+::: info
+In the following examples, we will use those fictional urls:  
+`https://my-pim.cloud.akeneo.com` for the PIM  
+`https://my-app.example.com/oauth/activate` for your App Activation URL  
+`https://my-app.example.com/oauth/callback` for your App Callback URL
+::::
+
+## Activation URL
+
+Your App must expose an activation URL.  
+When the PIM user sees your App in his PIM marketplace, he can click on the "Connect" button
+and he will be redirected to this activation URL with the PIM URL in the query.
+
+```
+
+https://my-app.example.com/oauth/activate?pim_url=https%3A%2F%2Fmy-pim.cloud.akeneo.com
+```
+
+## Authorization Request
+
+The user landed on your Activation URL and when you are ready to do so, you must start the Authorization Request.
+Like any other OAuth 2.0 application, you simply have to redirect the user to the Authorization Server (the PIM)
+with the following parameters:
+
+- `response_type` (required, must always be "code")
+- `client_id` (required)
+- `redirect_uri` (optional)
+- `scope` (optional)
+- `state` (recommended)
+
+```
+
+https://my-pim.cloud.akeneo.com/connect/apps/v1/authorize?
+    response_type=code&
+    client_id=[OAUTH_CLIENT_ID]&
+    redirect_uri=[CALLBACK_URL]&
+    scope=[REQUESTED_SCOPES]&
+    state=[STATE]
+```
+
+::: info
+You can consult the list of [availables scopes](https://help.akeneo.com).
+::::
+
+::: warning
+To protect your App from cross-site request forgery, you should send a random string in the `state` parameter.
+:::
+
+## Authorization Response
+
+The user will now have the opportunity to accept your authorization request and grant you the requested scopes.
+When he does so, he is redirected to the callback URL with the following parameters:
+
+```
+
+https://my-app.example.com/oauth/callback?
+    code=[AUTHORIZATION_CODE]&
+    state=[STATE]
```

Comment:
Shouldn't we return the authorized scopes here? The doc about it https://github.com/akeneo/pim-api-docs/pull/536/files#r752949045 should go there